test/rules/avoid_redundant_argument_values.dart

```diff
@@ -56,4 +58,9 @@ void main() {
   gg(1, 0); //LINT
   gg(1, 1); //OK
   gg(1); //OK
+
+  ggg(
+      1, // OK the  - first argument is required so that we can provide the second argument.
+      3);
+  ggg(1, 2); // LINT
```

Comment:
Consider putting the arguments on separate lines so that we can identify which one the lint should be associated with (here and elsewhere).

Reply:
Done.